src/sfizz/SfzFilter.h

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: BSD-2-Clause
+
+// This code is part of the sfizz library and is licensed under a BSD 2-clause
+// license. You should have receive a LICENSE.md file along with the code.
+// If not, contact the sfizz maintainers at https://github.com/sfztools/sfizz
+
+#pragma once
+#include <memory>
+
+namespace sfz {
+
+enum FilterType : int;
+
+/**
+   Multi-mode filter for SFZ v2
+   Available for mono and stereo. (NCh=1, NCh=2)
+
+   Parameters:
+     `cutoff`: it's the opcode `filN_cutoff` (Hz)
+     `q`: it's the opcode `filN_resonance` (dB)
+     `pksh`: it's the opcode `filN_gain` (dB)
+ */
+template <unsigned NCh>
+class Filter {
+public:
+    Filter();
+    ~Filter();
+
+    /**
+       Set up the filter constants.
+       Run it exactly once after instantiating.
```

Comment:
Is it true? It seems to me this could be changed live.

Reply:
This sets up only constants dependent on the sample rate.
The sample rate is not changed live. (or is it?)
Run it multiple times then, it should be fine.